keylime/cmd/convert_ima_policy.py

```diff
@@ -180,6 +190,12 @@ def main():
     elif args.ima_policy:
         with open(args.ima_policy, "r", encoding="utf8") as f:
             policy = json.load(f)
+        if not isinstance(policy, dict):
```

Comment:
Can this happen?

Reply:
It can probably happen if you construct a file that contains maybe a list for example.

Reply:
Indeed!